modules/bpi/bpi.images.inc

```diff
@@ -54,13 +55,15 @@ function bpi_assets_form($form, $form_state, $assets = array()) {
       '#title' => t('Available images'),
     );
 
-    foreach ($assets as $asset => $url) {
-      $form['bpi_assets'][$asset] = array(
-        '#type' => 'checkbox',
-        '#default_value' => 1,
-        '#prefix' => '<div class="bpi-syndicate-asset"><img src="' . $url . '" width="100" height="100" />',
-        '#suffix' => '</div>',
-      );
+    foreach ($assets as $type => $items) {
+      foreach ($items as $index => $asset) {
```

Comment:
It seems like the structure or the `$assets` parameter has been changed. It used to be a flat array of urls. It is now an array of types where each type is an array of assets of that type. Please update the DocBlock accordingly.